tools/openblas_support.py

```diff
@@ -8,13 +8,14 @@
 import textwrap
 import time
 import zipfile
+from packaging.tags import sys_tags
 
 from tempfile import mkstemp, gettempdir
 from urllib.request import urlopen, Request
 from urllib.error import HTTPError
 
-OPENBLAS_V = '0.3.18'
-OPENBLAS_LONG = 'v0.3.18'
+OPENBLAS_V = '0.3.20'
+OPENBLAS_LONG = 'v0.3.20'
```

Comment:
This will be a problem I think, we still have issues with newer OpenBLAS versions. IIRC we need either 0.3.18, or 0.3.21 + a macOS arm64 fix.

Reply:
There's multiple related issues here across numpy, scipy, openblas-libs, conda-forge and upstream openblas, that I can't all dig back up now.